src/widgets/modalChildren/NewConfirmModal.js

```diff
@@ -15,7 +15,7 @@ import {
   View,
   ViewPropTypes,
 } from 'react-native';
-
+import Modal from 'react-native-modalbox';
```

Comment:
The objective of this PR should be to remove this package from mSupply mobile completely.

Reply:
@Chris-Petty 
done. "397d519df"